config/base/crd/bases/olm.operatorframework.io_clusterextensions.yaml

```diff
@@ -474,15 +474,17 @@ spec:
                     - Reason: a string representation of the reason for the current state of the condition. Typically useful for building automation around particular Type+Reason combinations.
                     - Message: a human readable message that further elaborates on the state of the condition
 
-                  The current set of condition types are:
-                    - "Installed", represents whether or not the package referenced in the spec.packageName field has been installed
+                  The global set of condition types are:
+                    - "Installed", represents whether or not the a bundle has been installed for this ClusterExtension
                     - "Resolved", represents whether or not a bundle was found that satisfies the selection criteria outlined in the spec
-                    - "Deprecated", represents an aggregation of the PackageDeprecated, ChannelDeprecated, and BundleDeprecated condition types.
-                    - "PackageDeprecated", represents whether or not the package specified in the spec.packageName field has been deprecated
-                    - "ChannelDeprecated", represents whether or not the channel specified in spec.channel has been deprecated
-                    - "BundleDeprecated", represents whether or not the bundle installed is deprecated
                     - "Unpacked", represents whether or not the bundle contents have been successfully unpacked
 
+                  When the ClusterExtension is sourced from a catalog, the following conditions are also possible:
+                    - "Deprecated", represents an aggregation of the PackageDeprecated, ChannelDeprecated, and BundleDeprecated condition types
+                    - "PackageDeprecated", represents whether or not the package specified in the spec.source.catalog.packageName field has been deprecated
+                    - "ChannelDeprecated", represents whether or not any channel specified in spec.source.catalog.channels has been deprecated
+                    - "BundleDeprecated", represents whether or not the installed bundle is deprecated
+
                   The current set of reasons are:
                     - "Success", this reason is set on the "Unpacked", "Resolved" and "Installed" conditions when unpacking a bundle's content, resolution and installation/upgrading is successful
```

Comment:
Do we need to call out the reasons for the deprecated conditions?

Reply:
I don't think so. The "Reason" for these conditions are basically 1-to-1 with the Status. "The reason this is deprecated is because the author provided a deprecation"